torchtitan/train.py

```diff
@@ -474,9 +474,12 @@ def train(self):
             ft.maybe_semi_sync_training(
                 job_config,
                 ft_manager=self.ft_manager,
-                model=self.model_parts[0],
+                model_fragments=self.model_parts,
```

Comment:
Can we make the argument name consistent with `trainer` itself (mode_parts)? It is more consistent for the trainer if the naming conversion is done inside `FTManager`.